status/discover.go

```diff
@@ -60,18 +80,21 @@ func discoverUrlService(ns string, service string) (url string, err error) {
 	}
 	// If the client is not openshift return and avoid discover
 	if !client.IsOpenShift() {
+		log.Debugf("Kiali detect a no Openshift Cluster")
 		return "", nil
 	}
 	route, err := client.GetRoute(ns, service)
 	if err != nil {
+		log.Debugf("Kiali not detect the service %s in the namespace %s", service, ns)
 		return "", err
 	}
 
 	host := route.Spec.Host
-
 	if route.Spec.TLS != nil {
+		log.Debugf("Kiali detect the service %s in the namespace %s with SSL in %s", service, ns, "https://"+host)
 		return "https://" + host, nil
 	} else {
+		log.Debugf("Kiali detect the service %s in the namespace %s without SSL in %s", service, ns, "http://"+host)
```

Comment:
add the word "did" for slightly better English: "Kiali did not detect the service [%s] in the namespace [%s] without SSL in [%s]"